commit fe3dabb6a4b190e8dcdcfe70c9b6cb79eeae1768
Author: Bartosz Golaszewski <bgolaszewski@baylibre.com>
Date:   Fri Jan 11 18:21:18 2019 +0100

    ARM: dts: da850: fix interrupt numbers for clocksource
    
    commit e3966a766865da7ced1dece663697861dd5cf103 upstream.
    
    The timer interrupts specified in commit 3652e2741f42 ("ARM: dts:
    da850: Add clocks") are wrong but since the current timer code
    hard-codes them, the bug was never spotted.
    
    This patch must go into stable since, once we introduce a proper
    clocksource driver, devices with buggy device tree will stop booting.
    
    Fixes: 3652e2741f42 ("ARM: dts: da850: Add clocks")
    Cc: stable@vger.kernel.org
    Signed-off-by: Bartosz Golaszewski <bgolaszewski@baylibre.com>
    Signed-off-by: Sekhar Nori <nsekhar@ti.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/arch/arm/boot/dts/da850.dtsi b/arch/arm/boot/dts/da850.dtsi
index 47aa53ba6b92..559659b399d0 100644
--- a/arch/arm/boot/dts/da850.dtsi
+++ b/arch/arm/boot/dts/da850.dtsi
@@ -476,7 +476,7 @@
 		clocksource: timer@20000 {
 			compatible = "ti,da830-timer";
 			reg = <0x20000 0x1000>;
-			interrupts = <12>, <13>;
+			interrupts = <21>, <22>;
 			interrupt-names = "tint12", "tint34";
 			clocks = <&pll0_auxclk>;
 		};
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1333_fe3dabb6a4b190e8dcdcfe70c9b6cb79eeae1768.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1333_fe3dabb6a4b190e8dcdcfe70c9b6cb79eeae1768.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
